packages/TorneloScoresheet/src/components/Pin/Pin.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { SafeAreaView, Text } from 'react-native';
+import { styles } from './style';
+import {
+  CodeField,
+  Cursor,
+  useBlurOnFulfill,
+  useClearByFocusCell,
+} from 'react-native-confirmation-code-field';
+
+const CELL_COUNT = 4;
+
+const Pin = () => {
+  const [value, setValue] = useState('');
+  const ref = useBlurOnFulfill({ value, cellCount: CELL_COUNT });
+  const [props, getCellOnLayoutHandler] = useClearByFocusCell({
+    value,
+    setValue,
+  });
+
+  return (
+    <SafeAreaView style={styles.root}>
+      <CodeField
+        ref={ref}
+        {...props}
+        // Use `caretHidden={false}` when users can't paste a text value, because context menu doesn't appear
+        value={value}
+        onChangeText={setValue}
+        cellCount={CELL_COUNT}
+        rootStyle={styles.codeFieldRoot}
+        keyboardType="number-pad"
+        textContentType="oneTimeCode"
+        renderCell={({ index, symbol, isFocused }) => (
```

Comment:
should be `renderCell={({ index, symbol, isFocused } : {index: number, symbol: string, isFocused: boolean}) => (` to remove the error